Program.cs

```diff
@@ -3,16 +3,26 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Find all the anagrams and sort them based on the anagrams count from the words in the word list.
 // ---------------------------------------------------------------------------------------
 namespace Training {
    #region class Program --------------------------------------------------------------------------
+   /// <summary>Anagrams</summary>
    internal class Program {
       #region Method ------------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      static void Main () => Anagrams ();

```

Comment:
No need for separate method and the code in Anagrams method could have been inside Main itself.